packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -631,3 +632,7 @@ const getStyles = (theme: GrafanaTheme2) => {
     }),
   };
 };
+
+function pointDistance(a: Point, b: Point) {
+  return Math.hypot(a.x - b.x, a.y - b.y);
```

Comment:
```suggestion
  return Math.sqrt(Math.pow(a.x - b.x, 2), Math.pow(a.y - b.y, 2));
```

I'd suggest trying to stay away from `Math.hypot` as that can hurt the performance on some browsers (It's 2x slower on Safari).

Reply:
Like Torkel mentioned on slack, probably not a big concern for just a one off calculation, but it's good to know!